src/qt/bitcoingui.cpp

```diff
@@ -950,3 +955,14 @@ void BitcoinGUI::toggleHidden()
 {
     showNormalIfMinimized(true);
 }
+
+void BitcoinGUI::editConfig()
+{
+    QMessageBox mb;
+    mb.setWindowTitle(tr("Notice"));
+    mb.setText(tr("Any changes made to the configuration will require restarting the application."));
+    mb.exec();
+    QString configFile=QString::fromStdString(GetConfigFile().string());
+    //QDesktopServices::openUrl(QUrl(QString::fromStdString(GetConfigFile().string())));
+    QDesktopServices::openUrl(QUrl::fromLocalFile(configFile));
```

Comment:
Can this be changed back to avoid the need for the extra variable? We don't need the commented line either :D